### DISCUSSION TITLE
Cannot deploy through Github Pages

### DISCUSSION THREAD

**Discussion by ghost** [community/community#12903] [Other Feature Feedback, Questions, & Ideas]
Hello,

Suddenly when i am trying to use github pages, during deployment in actions tab i get this error and there is always a build failure:

**Billing owner (User or Organization) is deleted in GitHub.**

Can you explain how can i proceed? What the above message means?

Thanks,
Fanis

**Comment by gustavorps** [comment]
+1
Getting 404 for https://carapinagrande.github.io/
https://github.com/carapinagrande/carapinagrande.github.io/tree/gh-pages
https://github.com/carapinagrande/carapinagrande.github.io/runs/6249678507?check_suite_focus=true

**Comment by rhuid** [comment]
Is this issue resolved? Because I am facing the same problem.

**Comment by yoannchaudet** [comment]
This error does not sound familiar. Please open a support ticket.
